### PR TITLE
Include the controller verification for 3.7

### PR DESCRIPTION
**Pull request by e-minguez**
As part of the missing items from day 2 ops, I've fixed this for 3.7
@bfallonf PTAL

### CODE REVIEW

File: day_two_guide/topics/controller_role_verification.adoc [hunk 1]
```diff
@@ -12,31 +12,39 @@ hosts. The service runs in active/passive mode, meaning it should only be
...
 
 The {product-title} controllers execute a procedure to choose which host runs
-the service. The current running value is stored in the etcd cluster within the `/openshift.io/leases/controllers` key.
+the service. The current running value is stored in an annotation in a special
+`configmap` stored in the `kube-system` project.
 
 Verification of the master host actively running the
 `atomic-openshift-master-controllers.service` service is performed by running
-the following command on any of the master nodes:
+the following command as `cluster-admin`:
```
Comment (bfallonf) [inline day_two_guide/topics/controller_role_verification.adoc]: @e-minguez Can I suggest:

Verify the master host by running the `atomic-openshift-master-controllers.service` service as a `cluster-admin`:

File: day_two_guide/topics/controller_role_verification.adoc [hunk 2]
```diff
@@ -12,31 +12,39 @@ hosts. The service runs in active/passive mode, meaning it should only be
...
+`holderIdentity` property as:
 
 ----
-I1130 11:40:56.778373   30138 leaderelection.go:127] Attempting to acquire controller lease as master-8sn5gpzn, renewing every 30s
+master-<hostname>-<ip>-<8_random_characters>
 ----
 
-In order to find the identifiers of the master nodes, log in to each master node
-and run:
+In order to properly filter the hostname, the following command can be used:
```
Comment (bfallonf) [inline day_two_guide/topics/controller_role_verification.adoc]: Not too sure if just using "filter" is enough information. Perhaps:

Find the identifiers of the master host by filtering the hostname, using the following:

Does that adequately sum up what's happening?

**Comment by bfallonf** [comment]
@e-minguez Thanks so much for this! I added a couple comments (one suggestion and one question). Let me know what you think,

**Review by soukron — CHANGES_REQUESTED** [review]

File: day_two_guide/topics/controller_role_verification.adoc (hunk 1 above)
Comment (e-minguez) [reply day_two_guide/topics/controller_role_verification.adoc]: Done!

File: day_two_guide/topics/controller_role_verification.adoc (hunk 2 above)
Comment (e-minguez) [reply day_two_guide/topics/controller_role_verification.adoc]: I think I've fixed it, thx for the suggestion!

**Comment by e-minguez** [comment]
Thanks @bfallonf & @soukron for the suggestions. I've added those to the PR, PTAL

**Comment by bfallonf** [comment]
LGTM. Thanks @e-minguez . I'll merge this.

**Pull request merged**